e2e/miscellaneous/utilityBatch.test.ts

```diff
@@ -44,11 +43,10 @@ describe("Utility Batch Filtering", function () {
         const batchAll = ExtrinsicHelper.executeUtilityBatchAll(sender, badBatch);
         try {
             await batchAll.fundAndSend(fundingSource);
+            assert.fail("batchAll should have caused an error");
         } catch (err) {
-            error = err;
-            assert.notEqual(error, undefined, " batchAll should return an error");
+            assert.notEqual(err, undefined, " batchAll should return an error");
         }
-        assert.notEqual(error, undefined, " batchAll should return an error");
```

Comment:
This is the best way to do an error assertion.

Reply:
Just curious, any objection to using `assert.rejects`?
https://nodejs.org/api/assert.html#assertrejectsasyncfn-error-message

Also, I wonder if at some point (not now) we should switch these to Jest (as all of our other Node projects are using it for testing)?

Reply:
Or, if we want to keep using Mocha here, I'd be in favor of eventually swapping out the standard Node `assert` package for something a bit more capable. But definitely not a priority item.

Reply:
Ah right. Assert rejects likely would be better. I was just updating it to still match the existing pattern, but that would be better. Next time.

Reply:
Re: Jest

We might be able to, but I wasn't willing to make the switch yet.